include/sound/sof/header2.h

```diff
@@ -0,0 +1,410 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
```

Comment:
Can you make a little  bit more clear what is IPC4 introduced for?

- so it's a new version of IPC
- it is needed because we need to make the IPC compatible with Windows implemenation (?)
-

Reply:
Thanks @dbaluta. Moved the header2.h to ipc4/header.h and updated the commit message. I will work with my colleagues to improve the commit message.